apps/fluent-tester/src/TestComponents/Switch/Switch.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { SWITCH_TESTPAGE } from './consts';
+import { View, StyleSheet } from 'react-native';
+import { Switch } from '@fluentui-react-native/switch';
+
+const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 1,
+  },
+  square: {
+    backgroundColor: 'red',
+    width: 100,
+    height: 100,
+  },
+});
+
+const StandardUsage: React.FunctionComponent = () => {
+  return (
+    <View style={styles.container}>
+      <Switch defaultChecked={true} label={'Default Checked True'} />
+      <Switch defaultChecked={false} label={'Default Checked False'} />
+      <Switch defaultChecked={true} label={'Disabled Default Checked True'} disabled />
+      <Switch defaultChecked={false} label={'Disabled Default Checked False'} disabled />
+    </View>
+  );
+};
+
+const OnChangeUsage: React.FunctionComponent = () => {
+  const defaultDisplaySquare = true;
+  const [displaySquare, setDisplaySquare] = React.useState(defaultDisplaySquare);
+
+  const toggleSquare = (checked) => {
+    setDisplaySquare(checked);
+  };
+
+  return (
+    <View style={styles.container}>
+      <Switch defaultChecked={defaultDisplaySquare} onChange={toggleSquare} />
```

Comment:
Does `toggleSquare` do anything here if the value of checked is never passed?

I would rewrite this as `<Switch defaultChecked={displaySquare} onChange={() => setDisplaySquare(!displaySquare)} />`

Reply:
Yeah `toggleSquare` wouldn't do anything if the checked value was never passed, but the job of onChange is to pass the updated checked value. So we don't need to worry about.

The `defaultChecked` prop is only for setting the inital checked state. Users do not have to worry about managing the checked state. The checked state is managed by the control. If the users would like to manage the checked state themselves they would need to to use the `checked` prop instead.